package.json

```diff
@@ -124,6 +124,9 @@
             "type": "go",
             "request": "launch",
             "mode": "debug",
+			"remotePath": "",
```

Comment:
Here and elsewhere whitespace appears to be off.  The project uses tabs.  Could you make sure whitespace and formatting match?